host-interaction/gui/enumerate-gui-resources.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: enumerate gui resources
+    namespace: host-interaction/gui
+    author: johnk3r
+    scope: function
+    att&ck:
+      - Discovery::Application Windows Discovery [T1010]
```

Comment:
typo, should be `Application Window Discovery`, see https://attack.mitre.org/techniques/T1010/